### PR TITLE
feat(account-sidebar, dedicated): change of url to status-ovhcloud

### PR DESCRIPTION
**Pull request by anooparveti**
Change of URL from travaux to status-ovhcloud
 ref: MANAGER-6669

Signed-off-by: Anoop <anooparveti@gmail.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-6669
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

Change of URL from Travaux to status-ovhcloud

### CODE REVIEW

**Review by radireddy — APPROVED** [review]

**Review by varun257 — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/config/constants.config.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const constants = {
     RENEW_URL:
       'https://eu.ovh.com/cgi-bin/order/renew.cgi?domainChooser={serviceName}',
     vmsUrl: 'http://travaux.ovh.net/vms/',
-    travauxUrl: 'http://travaux.ovh.net/',
+    travauxUrl: 'https://www.status-ovhcloud.com/',
```
Comment (antleblanc) [inline packages/manager/apps/dedicated/client/app/config/constants.config.js:8]:
```suggestion
    statusUrl: 'https://www.status-ovhcloud.com/',
```

Comment (antleblanc) [reply packages/manager/apps/dedicated/client/app/config/constants.config.js:8]: Should be reflected here:

```txt
packages/manager/apps/dedicated/client/app/app.module.js
packages/manager/apps/dedicated/client/app/config/constants.config.js
packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.controller.js
```

File: packages/manager/apps/dedicated/client/app/config/constants.config.js [hunk 2]
```diff
@@ -747,7 +747,7 @@ const constants = {
     RENEW_URL:
       'https://ca.ovh.com/cgi-bin/order/renew.cgi?domainChooser={serviceName}',
     vmsUrl: 'http://status.ovh.net/vms/',
-    travauxUrl: 'http://status.ovh.net/',
+    travauxUrl: 'https://www.status-ovhcloud.com/',
```
Comment (antleblanc) [inline packages/manager/apps/dedicated/client/app/config/constants.config.js:750]:
```suggestion
    statusUrl: 'https://www.status-ovhcloud.com/',
```

**Review by antleblanc — CHANGES_REQUESTED** [review]

**Review by radireddy — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.controller.js [hunk 3]
```diff
@@ -45,7 +45,7 @@ export default class DedicatedServerDashboard {
     this.servicesStateLinks = {
       weathermap: WEATHERMAP_URL[this.user.country],
       vms: this.constants.vmsUrl,
-      travaux: this.constants.travauxUrl,
+      travaux: this.constants.statusUrl,
```
Comment (antleblanc) [inline packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.controller.js:48]:
```suggestion
      status: this.constants.statusUrl,
```

**Review by antleblanc — CHANGES_REQUESTED** [review]

**Review by radireddy — APPROVED** [review]

**Review by mohammed-zahaf — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

**Comment by antleblanc** [comment]
@anooparveti Could you please rebase it against `master` branch?
Thank you!

**Review by antleblanc — APPROVED** [review]

**Pull request merged**